api/channel.go

```diff
@@ -1142,3 +1146,12 @@ func updateNotifyProps(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 }
+
+func GetChannelMember(channelId, userId string) (model.ChannelMember, *model.AppError) {
+	result := <-Srv.Store.Channel().GetMember(channelId, userId)
+	if result.Err != nil {
+		return model.ChannelMember{}, result.Err
```

Comment:
shouldn't this be `return nil, result.Err` ?

also if you're creating this function why not replace the other calls to ChannelStore GetMember by this function around the code?